src/map/utils/petutils.cpp

```diff
@@ -1718,6 +1728,19 @@ namespace petutils
                 PPet->addModifier(Mod::MDEF, PChar->PMeritPoints->GetMeritValue(MERIT_FINE_TUNING, PChar));
             }
         }
+        else if (PPet->getPetType() == PETTYPE_LUOPAN && PMaster->objtype == TYPE_PC)
+        {
+            PPet->SetMLevel(PMaster->GetMLevel());
+            PPet->addModifier(Mod::DMG, -50);
```

Comment:
Damage mod should probably be added in the Lua global too, if only so people who want to tinker with bubbles can stay in Lua Land.

Reply:
Agreed!